src/main/Main.ts

```diff
@@ -1,12 +1,50 @@
-import {app, ipcMain, nativeImage, BrowserWindow, screen} from "electron";
+import {app, nativeImage, BrowserWindow, screen, globalShortcut, Menu} from "electron";
 import {readFileSync} from "fs";
 import {windowBoundsFilePath} from "../utils/Common";
+import {buildMenuTemplate} from "./Menu";
+
+/**
+ * Contains references to all currently open windows.
+ *
+ * @type {Array}
+ */
+let windows: Electron.BrowserWindow[] = [];
+
+/**
+ * The index of the currently focussed window in `windows`.
+ *
+ * @type {number}
```

Comment:
@jsphpl Most of the codes [out there](https://github.com/Microsoft/vscode/blob/master/src/vs/editor/contrib/folding/common/indentFoldStrategy.ts#L16-L18) simply use plain `/** ... */` comments. But you can have `@param` and stuff too so like:

<img width="576" alt="screen shot 2016-12-11 at 4 37 18 pm" src="https://cloud.githubusercontent.com/assets/2157285/21080338/8f7db8bc-bfc1-11e6-9d34-e1466f35aa0a.png">

But it's for just when you really need to specify the each param so you can have hover options. Most of the times you just go with a simple description of the function and because TypeScript is aware of the types it generates you a perfectly well explained scheme of the function:

<img width="620" alt="screen shot 2016-12-11 at 4 43 07 pm" src="https://cloud.githubusercontent.com/assets/2157285/21080346/ad47088a-bfc1-11e6-9dd0-f36f54344ee8.png">

Also you can have markdown: 

<img width="545" alt="screen shot 2016-12-11 at 4 50 07 pm" src="https://cloud.githubusercontent.com/assets/2157285/21080356/ee38efde-bfc1-11e6-95d3-7476dea90510.png">

So you can just bold or italic the params and type faster and more clear